src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,50 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
-);
+export const MovieCard = ({ movie }) => {
+  const { imgUrl, title, description, imdbUrl } = movie;
+
+  return (
+    <>
```

Comment:
A fragment should be used only when the component does not return everything wrapped in a single tag.